### ISSUE TITLE
ACVP Specification Issue for CMAC-TDES

### ISSUE THREAD

**Issue by jbarnesweb** [usnistgov/ACVP-Server#106]
https://pages.nist.gov/ACVP/draft-fussell-acvp-mac.html#name-cmac-tdes

Capabilities file included this object.

```
			{
				"algorithm": "CMAC-TDES",
				"revision": "1.0",
				"capabilities": [
					{
						"direction": [ "gen", "ver" ],
						"keyingOption": [ 1 ],
						"msgLen": [
							{
								"min": 0,
								"max": 65536,
								"increment": 8
							}
						],
						"macLen": [
							{
								"min": 32,
								"max": 64,
								"increment": 8
							}
						]
					}
				]
			},

```

I also tried with this object.

```
      {
        "algorithm": "CMAC-TDES",
        "revision": "1.0",
        "capabilities": [
          {
            "direction": [ "gen", "ver" ],
            "keyingOption": [ 1 ],
            "keyLen": [ 192 ],
            "msgLen": [
              {
                "min": 0,
                "max": 65536,
                "increment": 8
              }
            ],
            "macLen": [
              {
                "min": 32,
                "max": 64,
                "increment": 8
              }
            ]
          }
        ]
      },

```

Vector sets included this testGroup/testCase.

```

    "testGroups": [
      {
        "direction": "gen",
        "keyLen": 0,
        "keyingOption": 1,
        "macLen": 32,
        "msgLen": 0,
        "testType": "AFT",
        "tests": [
          {
            "key": "E980AED38F9EBA67AE3210528F68495E23B564642CFDE50D",
            "key1": "E980AED38F9EBA67",
            "key2": "AE3210528F68495E",
            "key3": "23B564642CFDE50D",
            "message": "",
            "tcId": 1
          },


```

Why is there a keyLen: 0 in the testGroup and not in the capabilities spec at the link? Would you correct the spec, please?

**Comment by Kritner** [comment]
The `keyLen` property shouldn't be included in the delivered prompt file.  It is always 0, and can be ignored.  I'll get it removed from the produced files in the next update.

**Comment by Kritner** [comment]
This change is on demo in release [v1.1.0.18](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.18)

**Comment by Kritner** [comment]
This change has been released to prod in release [v1.1.0.18](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.18).
